contrib/freshmaker-cli

```diff
@@ -141,6 +146,16 @@ def get_api_url(deployment, env):
 
     return envs[env]
 
+def get_errata_api(env):
```

Comment:
This is a misleading function name since it returns the API for the custom filter you've created. Could you please rename this to make this more clear?

Also, could you please describe what the filter does in the docstring of this function?